src/MizarWidgets/SamplingWithFrameTrackWidget.cpp

```diff
@@ -32,19 +41,28 @@ void SamplingWithFrameTrackWidget::Init(
 
 SamplingWithFrameTrackWidget::~SamplingWithFrameTrackWidget() = default;
 
-[[nodiscard]] Baseline<SamplingWithFrameTrackInputWidget*>
-SamplingWithFrameTrackWidget::GetBaselineInput() const {
+Baseline<SamplingWithFrameTrackInputWidget*> SamplingWithFrameTrackWidget::GetBaselineInput()
+    const {
   return Baseline<SamplingWithFrameTrackInputWidget*>(ui_->baseline_input_);
 }
 
-[[nodiscard]] Comparison<SamplingWithFrameTrackInputWidget*>
-SamplingWithFrameTrackWidget::GetComparisonInput() const {
+Comparison<SamplingWithFrameTrackInputWidget*> SamplingWithFrameTrackWidget::GetComparisonInput()
+    const {
   return Comparison<SamplingWithFrameTrackInputWidget*>(ui_->comparison_input_);
 }
 
-const Baseline<QString> SamplingWithFrameTrackWidget::kBaselineTitle =
-    Baseline<QString>(QStringLiteral("Baseline"));
-const Comparison<QString> SamplingWithFrameTrackWidget::kComparisonTitle =
-    Comparison<QString>(QStringLiteral("Comparison"));
+void SamplingWithFrameTrackWidget::OnSignificanceLevelSelected(int index) {
+  constexpr int kIndexOfFivePercent = 0;
+  significance_level_ =
+      (index == kIndexOfFivePercent) ? kDefaultSignificanceLevel : kAlternativeSignificanceLevel;
```

Comment:
I have been thinking about a similar suggestion here, as for the multiplicity correction, but I don't have a good solution. 
You could use `QCombobox::setItemData(int index, const QVariant &value)` and `QCombobox::itemData(int index)`, but I dont know if its worth it (probably not).